homeassistant/helpers/state.py

```diff
@@ -120,6 +120,32 @@ def get_changed_since(states, utc_point_in_time):
             if state.last_updated >= utc_point_in_time]
 
 
+def async_get_service(hass, service_domain, state):
```

Comment:
I'm hitting cyclic imports if I move the function together with the static dicts it uses to the service helper. I could move only the function and pass the dicts as arguments, but that looks pretty bad.

This is a sign that the current structure with the static dicts that needs loads of imported strings from components is unstable. Helpers should not import from components. The only other helpers that does that are `condition` and `restore_state`.